pkg/cluster/k8sres.go

```diff
@@ -332,17 +332,16 @@ func generateVolumeMounts() []v1.VolumeMount {
 	}
 }
 
-func generateSpiloContainer(
-	name string,
+func (c *Cluster) generateSpiloContainer(
```

Comment:
Why it suddenly became cluster dependent, only for logging purposes? I think it would be better to log `spiloEnvVars` directly after deduplication right in the `generateStatefulSet`

Reply:
Thanks @erthalion - I just thought personally it would be useful to have logging code in `generateSpiloContainer`, but I can do as you suggest instead.